connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerInfo.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.connect.runtime;
+
+import org.apache.kafka.common.utils.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.management.ManagementFactory;
+import java.lang.management.RuntimeMXBean;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * Connect Worker system and runtime information.
+ */
+public class WorkerInfo {
+    private static final Logger log = LoggerFactory.getLogger(WorkerInfo.class);
+    private static final RuntimeMXBean RUNTIME;
+
+    static {
+        RUNTIME = ManagementFactory.getRuntimeMXBean();
+    }
+
+    private final Map<String, Object> values;
+
+    /**
+     * Constructor.
+     */
+    public WorkerInfo() {
+        this.values = new LinkedHashMap<>();
+        addRuntimeInfo();
+    }
+
+    /**
+     * Collect general runtime information.
+     */
+    protected void addRuntimeInfo() {
+        List<String> jvmArgs = RUNTIME.getInputArguments();
+        values.put("jvm.args", jvmArgs);
+    }
+
+    private static Object format(Object value) {
+        return value == null ? "NA" : value;
+    }
+
+    /**
+     * Log the values of this object at level INFO.
+     */
+    // Equivalent to logAll in AbstractConfig
+    public void logAll() {
+        StringBuilder b = new StringBuilder();
+        b.append(getClass().getSimpleName());
+        b.append(" values: ");
+        b.append(Utils.NL);
```

Comment:
Why not `System.lineSeparator()` instead of `Utils.NL`?

Reply:
It's literally taken as is from `AbstractConfig`. Also, `Utils.NL` is indeed equivalent (`System.getProperty("line.separator");`)